flow-server/src/main/java/com/vaadin/flow/server/startup/LookupServletContainerInitializer.java

```diff
@@ -74,7 +74,7 @@ public void process(Set<Class<?>> classSet, ServletContext servletContext)
                     + LookupInitializer.class + " always presents "
```

Comment:
This line also sounds bork.
```suggestion
                    + LookupInitializer.class + " should always be present "
```